fzf.el

```diff
@@ -244,6 +244,25 @@ If nil, fzf prompts have the same history in all modes."
   :safe #'booleanp
   :group 'fzf)
 
+(defcustom fzf/position-bottom-of-frame nil
+  "Set the position of the fzf to always be at the bottom of the frame."
+  :type 'boolean
+  :safe #'booleanp
+  :group 'fzf)
+
+(defun fzf/position-bottom-of-frame-watcher (symbol newval operation where)
```

Comment:
I would also like you to move the definition of the watcher function and the call to `add-variable-watcher` to a section of code located *after* the `Internal variables` section and at the beginning of the `Internal helper function' section (which currently is not named as such and should be).

Then a question: do you consider the `fzf/position-bottom-of-frame-watcher` part of the *main* interface of the package? I don't think so ,right?   It's not an interactive command and is not meant to be invoked by the user.  Therefore I would rename it to  something that makes it look like an internal function, something like: ``fzf--position-bottom-of-frame-watcher`

Reply:
Yeah, obviously. fixed in e3f88ee